src/peergos/shared/user/UserContext.java

```diff
@@ -30,6 +30,7 @@
     public final SigningKeyPair signer;
     public final BoxingKeyPair boxer;
     public final Fragmenter fragmenter;
+    public final Set<String> sharedFiles;
```

Comment:
Would be good to know what these strings are. Ideally with a type. Maybe even a class to wrap the whole set too.